File: Readio/src/main/java/ninegle/Readio/user/repository/BlackListRepository.java
package ninegle.Readio.user.repository;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import ninegle.Readio.user.domain.BlackList;

@Repository
public interface BlackListRepository extends JpaRepository<BlackList, Long> {
	Optional<BlackList> findByInversionAccessToken(String token);

}

File: Readio/src/main/java/ninegle/Readio/user/repository/BlackListRepository.kt
package ninegle.Readio.user.repository

import ninegle.Readio.user.domain.BlackList
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface BlackListRepository : JpaRepository<BlackList, Long> {
    fun findByInversionAccessToken(token: String): BlackList?
}
